### ISSUE TITLE
`--disable-wallet` has no effect

### ISSUE THREAD

**Issue by jamilbk** [dashpay/dash#1732]
Hi there,

I'm trying to build dash from source. I don't need wallet functionality, only `dashd` with RPC access.

I'm getting this when trying to build with `./configure --disable--wallet` and 0.12.2.1:

```
In file included from ./wallet/walletdb.h:10:0,
                 from wallet/wallet.h:20,
                 from activemasternode.h:10,
                 from init.cpp:48:
./wallet/db.h:21:20: fatal error: db_cxx.h: No such file or directory
compilation terminated.
Makefile:5657: recipe for target 'libbitcoin_server_a-init.o' failed
make[2]: *** [libbitcoin_server_a-init.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/dash-build/dash-0.12.2.1/src'
Makefile:9434: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/dash-build/dash-0.12.2.1/src'
make: *** [all-recursive] Error 1
Makefile:656: recipe for target 'all-recursive' failed
ERROR: Service 'dash' failed to build: The command '/bin/sh -c make -j$(nproc)' returned a non-zero code: 2
```

It seems like dash is still looking for libdb4.8++. Is this expected behavior? Are you guys able to reproduce?

**Comment by schinzelh** [comment]
It‘s a known issue, Dash wallet can’t be built without wallet functionality due to unresolved includes. If you want to disable wallet you can do it during runtime, using wallet=0 in dash.conf

**Comment by UdjinM6** [comment]
Duplicate of #1557 #1357

Yes, it's not currently possible to build with `--disable-wallet` but you can disable it afterwards like @schinzelh suggested above. Small correction though: it's `-disablewallet` in cmdline or `disablewallet=1` in `dash.conf`.

**Comment by UdjinM6** [comment]
Should be fixed by #1733.
